_ml-commons-plugin/model-serving-framework.md

```diff
@@ -68,6 +68,13 @@ Field | Data type | Description
 | `framework_type` | string  | The framework the model is using. Currently, we support `sentence_transformers` and `huggingface_transformers` frameworks. The `sentence_transformers` model outputs text embeddings directly, so ML Commons does not perform any post processing. For `huggingface_transformers`, ML Commons performs post processing by applying mean pooling to get text embeddings. See the example [`all-MiniLM-L6-v2` Huggingface model](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2) for more details. |
 | `all_config` _(Optional)_ | string | This field is used for reference purposes. You can specify all model configurations in this field. For example, if you are using a Huggingface model, you can minify the `config.json` file to one line and save its contents in the `all_config` field. Once the model is uploaded, you can use the get model API operation to get all model configurations stored in this field. |
 
+When using a pre-trained model, the following fields are required in the `model_config` object.
```

Comment:
These two parameters are optional, read this doc https://github.com/opensearch-project/ml-commons/blob/2.x/docs/model_serving_framework/text_embedding_model_examples.md#11-trace-sentence-transformers-model

> Sentence transformer model already includes post-processing logic. So no need to specify pooling_mode/normalize_result when upload model.

> Without [sentence-transformers](https://pypi.org/project/sentence-transformers/) installed, you can trace this model AutoModel.from_pretrained('sentence-transformers/all-MiniLM-L6-v2'). But model traced this way doesn't include post-processing. So user have to specify post-process logic with pooling_mode and normalize_result.